docs/documentation.md

```diff
@@ -17,6 +17,124 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    <td class="centre"><a href="https://www.rabbitmq.com/connections.html">Connections</a></td>
+  </tr>
+   <tr>
+   <td class="centre">Publish and Consume Messages </td>
+    <td class="centre">
+    <ul>
+      <li><a href="./publishers.md">Publishing Messages</a></li>
+      <li><a href="./publishers.md">Consuming Message</a></li>
+      <li><a href="./confirms.md">Consumer Acknowledgements and Publisher Confirms</a></li>
+    </ul>
+   </td>
+    <tr>
+<tr>
+    <td class="centre">Choose and configure the topology</td>
+    <td class="centre"><ul>
+      <li><a href="https://www.rabbitmq.com/tutorials/amqp-concepts#exchanges">Exchange and Exchange Types</a></li>
+      <li><a href="https://www.rabbitmq.com/docs/queues">Choose the Queue</a>/<a href="https://www.rabbitmq.com/docs/streams">Stream</a></li>
+    </ul>
+    </td>
+    <tr>
+    <tr>
+    <td class="centre">Choose and configure data safety mechanisms to ensure messages are always delivered </td>
+    <td class="centre"><a href="https://www.rabbitmq.com/docs/reliability">Reliability</a></td>
+  </tr>
+    <tr>
+    <td class="centre">Depending on your requirements, you will also need to complete other RabbitMQ configuration described in these sections</td>
+    <td class="centre"><ul>
+      <li>How to use RabbitMQ</li>
+      <li>How to Manage RabbitMQ</li>
+      <li><a href="https://www.rabbitmq.com/docs/monitoring">How to Monitor RabbitMQ</a></li>
+    </ul></td>
+  </tr>
+  <tr>
+    <td class="centre">Troubleshoot RabbitMQ</td>
+    <td class="centre"><ul>
+      <li><a href="https://www.rabbitmq.com/docs/troubleshooting-networking">Troubleshooting Network Connectivity</a></li>
+      <li><a href="https://www.rabbitmq.com/docs/troubleshooting-ssl">Troubleshooting TLS</a></li>
+      <li><a href="https://www.rabbitmq.com/docs/troubleshooting">Troubleshooting RabbitMQ</a></li>
+    </ul></td>
+  </tr>
+  </tr>
+</table> 
+
+
+
+## User: Administrator
+
+**Important:** The following table is a generic list of tasks an administrator would follow to use RabbitMQ in a production environment. It is important to note that every production environment is unique so the installation and setup route an administrator takes is dependent on that. 
+
+<table id="administrator">
+   <tr>
+    <th class="centre">Task</th>
+    <th class="centre">Documentation</th>
+  </tr>
+   <tr>
+    <td class="centre">Before deploying RabbitMQ in a production environment, take a look at our deployment guidelines</td>
+    <td class="centre"><a href="https://www.rabbitmq.com/docs/download">Deployment Guidelines</a></td>
+  </tr>
+  <tr>
+  <tr>
+    <td class="centre">Choose an installation option for your platform. When you are deploying RabbitMQ, choose one of the latest RabbitMQ and Erlang versions.</td>
+    <td class="centre"><a href="https://www.rabbitmq.com/docs/download">Installing RabbitMQ</a> and <a href="https://www.rabbitmq.com/docs/platforms">Supported Platforms</a></td>
```

Comment:
"Supported Platforms" should be linked from the "Installing RabbitMQ" page. The "supported" notion is implicit in "installing": the user would click to learn how to install and will look for their platform in that page.